lib/ace/ext/papeeria/katex-demo.coffee

```diff
@@ -0,0 +1,31 @@
+define (require, exports, module) ->
+    require "ace/ext/jquery"
+
+    $(document).ready ->
+        # Adding KaTeX CSS
+        cssKatexPath = require.toUrl "../katex/katex.min.css"
+        linkKatex = document.createElement "link"
+        linkKatex.setAttribute "rel", "stylesheet"
+        linkKatex.setAttribute "href", cssKatexPath
+        $("head")[0].appendChild linkKatex
```

Comment:
If you're sure you have 1 head element, you can just use `$("head").append(linkKatex)`